assignments/prototypes.js

```diff
@@ -15,13 +17,31 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(gameObjectAttributes) {
+  this.createdAt = gameObjectAttributes.createdAt;
+  this.dimensions = {length: gameObjectAttributes.length,
+                     width: gameObjectAttributes.width,
```

Comment:
Remember the length, width, and height do not exist in the `gameObjectAttributes` directly but are a layer deep in the dimensions. So try something like:
```
length: gameObjectAttributes.dimensions.length
```